clients/python-wrapper/lakefs/client.py

```diff
@@ -0,0 +1,117 @@
+import base64
+import binascii
+from typing import Optional
+import requests
+from requests.auth import HTTPBasicAuth
+
+import lakefs_sdk
+from lakefs_sdk.client import LakeFSClient
+
+from lakefs.config import ClientConfig
+from lakefs.exceptions import NoAuthenticationFound, UnsupportedOperationException
+
+
+class Client:
+    """
+    Wrapper around lakefs_sdk's client object
+    Takes care of instantiating it from the environment
+    """
+
+    _client: Optional[LakeFSClient] = None
+    _http_client: requests.Session = None
```

Comment:
```suggestion
    _http_client: Optional[requests.Session] = None
```

For all values that are type hinted and can be `None` please..